nncf/torch/dynamic_graph/io_handling.py

```diff
@@ -329,7 +327,22 @@ def wrap_inputs(self, model_args: Tuple, model_kwargs: Dict) -> Tuple[Tuple, Dic
                 bound_model_params.apply_defaults()
 
             potential_tensor = bound_model_params.arguments[param_name]
+            if not isinstance(dummy_input, torch.Tensor):
+                if isinstance(potential_tensor, torch.Tensor):
+                    raise RuntimeError(
```

Comment:
It looks like, this change is not backwards compatible.

Reply:
Do you suggest to remove runtime error then?

Reply:
Such combination alters NNCFNetwork building and the result of additional unexpected nncf_input could cause an error in an compression algo. In case we have such combination (which is unlikely) in some model, it worth investigation on how it actually works

Reply:
Ok, done